the-collab-lab/tcl-6-smart-shopping-list
# Es pja add new item

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



## Related Issue

Closes #6 

## Acceptance Criteria

- User is presented with a form that allows them to enter the name of the item and select how soon they anticipate needing to buy it again (Soon, Kind of soon, Not soon)
- When the user submits the form, the item is saved to the database, associated with the user’s token
- Along with the item name, an integer corresponding to the estimated number of days until next purchase is saved: 7 for “Soon”, 14 for “Kind of soon”, and 30 for “Not soon”
- The last purchased date should be set to null initially (i.e. you can create an item without purchasing it)

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![Image 4-17-20 at 4 37 PM (1)](https://user-images.githubusercontent.com/49720658/79628027-233f1880-80f2-11ea-9775-84c0b11eff0e.jpg)


### After

![Image 4-17-20 at 4 29 PM](https://user-images.githubusercontent.com/49720658/79628032-3356f800-80f2-11ea-969d-494cdf80eb2a.jpg)


## Testing Steps / QA Criteria

1)  Git pull the branch using `git pull origin es-pja-add-new-item` and `git checkout es-pja-add-new-item` to switch to the branch.
2) Run `npm install` to add dependencies, then `npm start` to run the app.
3) When the app runs in the browser, you will be presented with a page titled "Shopping List" with two buttons "Shopping" and "Add Item."
4) Clicking "Add Item" should present a form with a "Name" input, a Schedule option with a dropdown menu and 3 choices, and a Submit button.
